Gems/AWSCore/Code/Source/ResourceMapping/AWSResourceMappingManager.cpp

```diff
@@ -118,6 +118,11 @@ namespace AWSCore
         }, resourceKeyName);
     }
 
+    bool AWSResourceMappingManager::HasResourceType(const AZStd::string& resourceKeyName) const
```

Comment:
`HasResourceType` sounds like you are providing the type which would match the `attribute.resourceType`, but you are passing in a `resourceKeyName` which does not appear to be a `resourceType`, what if this function was simply called `HasResource`?

Reply:
Yes, I like it! Fixed :D